expertsystem/data.py

```diff
@@ -301,13 +301,13 @@ def compute_isospin_projection(  # pylint: disable=too-many-arguments
         )
 
 
-class ParticleCollection(abc.Mapping):
-    """Safe, `dict`-like collection of `.Particle` instances."""
+class ParticleCollection(abc.Set):
```

Comment:
Shouldn't it be a `MutableSet`. I think support for the deletion or update of elements would also be convenient (currently there is only merge).

Reply:
Yes that is true. And then probably the `add` method makes more sense as well (it's in the `MutableSet` base class).

Reply:
30de4cdc and a41bb834